proposals/10-egress-gateways.md

```diff
@@ -166,15 +166,14 @@ spec:
     type: FQDN
     fqdn:
       hostname: api.openai.com
-      port: 443
-  tls:
-    mode: Terminate | Passthrough | Mutual
-    sni: api.openai.com
-    caBundleRef:
-      name: vendor-ca
-    # clientCertificateRef:  # if MUTUAL
-    #   name: egress-client-cert
-  # possible extension semantics, for illustration purposes only.
+    ports:
+    - number: 443
+      protocol: TLS
+      tls:
+        mode: SIMPLE | MUTUAL | PASSTHROUGH | PLATFORM_PROVIDED | INSECURE_DISABLE
+        sni: api.openai.com
+        caBundleRef:
+          name: vendor-ca
```

Comment:
This is not related, but i wonder where is the caBundle located, just a name is not enough

Reply:
Ah yeah it's not GA yet so we can't make it the default